stripe/src/main/java/com/stripe/android/StripePaymentController.kt

```diff
@@ -146,7 +153,12 @@ internal class StripePaymentController internal constructor(
             withContext(Dispatchers.Main) {
                 result.fold(
                     onSuccess = { intent ->
-                        handleNextAction(host, intent, requestOptions)
```

Comment:
should we extract `returnUrl` from the response `intent` and default to `ConfirmStripeIntentParams.DEFAULT_RETURN_URL` otherwise?